src/imgui/ImGuiReverseBar.cc

```diff
@@ -161,27 +164,24 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 		const auto& reverseManager = motherBoard->getReverseManager();
 		bool reverseEnabled = reverseManager.isCollecting();
 
-		im::Menu("Load replay ...", reverseEnabled, [&]{
-			ImGui::TextUnformatted("Select replay"sv);
-			im::ListBox("##select-replay", [&]{
-				struct Names {
-					Names(std::string f, std::string d) // workaround, needed for clang, not gcc or msvc
-						: fullName(std::move(f)), displayName(std::move(d)) {} // fixed in clang-16
-					std::string fullName;
-					std::string displayName;
-				};
-				std::vector<Names> names;
+		loadReplayOpen = im::Menu("Load replay ...", reverseEnabled, [&]{
+			if (!loadReplayOpen) {
```

Comment:
Same comment about `isOpen` vs `wasOpen`.

Reply:
Please resolve the conversation if you're satisfied.